app_dart/lib/src/request_handlers/get_build_status.dart

```diff
@@ -26,14 +27,15 @@ class GetBuildStatus extends RequestHandler<Body> {
   final DatastoreServiceProvider datastoreProvider;
   final BuildStatusServiceProvider buildStatusProvider;
 
-  static const String branchParam = 'branch';
+  static const String kRepoParam = 'repo';
 
   @override
   Future<Body> get() async {
     final DatastoreService datastore = datastoreProvider(config.db);
     final BuildStatusService buildStatusService = buildStatusProvider(datastore);
-    final String branch = request!.uri.queryParameters[branchParam] ?? 'master';
-    final BuildStatus status = (await buildStatusService.calculateCumulativeStatus(branch: branch))!;
+    final String repoName = request!.uri.queryParameters[kRepoParam] ?? 'flutter';
```

Comment:
For migration purposes, we need to default to flutter/flutter otherwise the build dashboard would fail (there's a few handlers that will need to be updated)